plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestingPostgreSqlServer.java

```diff
@@ -146,4 +167,27 @@ public void close()
     {
         dockerContainer.close();
     }
+
+    public static class DatabaseEvents
+    {
+        private final TestingPostgreSqlServer server;
+        private final int currentEventsCount;
+
+        private DatabaseEvents(TestingPostgreSqlServer server)
+        {
+            this.server = requireNonNull(server, "server is null");
+            currentEventsCount = server.getRemoteDatabaseEvents().size();
+        }
+
+        public Stream<String> streamQueriesContaining(String queryPart, String... alternativeQueryParts)
```

Comment:
What do you mean by `alternative`? Maybe just `queryParts`?

Reply:
it's because we are doing `query contains (queryPart || alternativeQueryParts[0] || alternativeQueryParts[1] || alternativeQueryParts[2])` which is different from `query contains (queryPart && alternativeQueryParts[0] && alternativeQueryParts[1] && alternativeQueryParts[2])` and I thought that it's valid to give this hint in method definition. Still It's probably better to rename the method into `streamQueriesContainingAnyOf`